src/views/InvolvementProfile/components/Membership/components/MemberList/components/MemberListItem/index.js

```diff
@@ -311,19 +358,28 @@ const MemberListItem = ({
 
     content = (
       <>
-        <Grid container style={rowStyle} direction="row" alignItems="center">
-          <Grid item xs={4} style={rowStyle}>
+        <Grid container alignItems="center" spacing={2} wrap="nowrap">
+          <Grid item md={1} style={rowStyle}>
+            <IMG
```

Comment:
As mentioned above, using `Avatar` makes more sense than using the `react-graceful-image` component here.